frontend/src/metabase/admin/performance/strategies.ts

```diff
@@ -1,38 +0,0 @@
-import { t } from "ttag";
-import * as Yup from "yup";
-
-import { DurationUnit } from "metabase-types/api";
-
-const durationUnits = new Set(Object.values(DurationUnit).map(String));
-
-const positiveInteger = Yup.number()
-  .positive(t`Enter a positive number.`)
-  .integer(t`Enter an integer.`);
-
-export const inheritStrategyValidationSchema = Yup.object({
-  type: Yup.string().equals(["inherit"]),
-});
-
-export const doNotCacheStrategyValidationSchema = Yup.object({
-  type: Yup.string().equals(["nocache"]),
-});
-
-export const defaultMinDurationMs = 1000;
-export const adaptiveStrategyValidationSchema = Yup.object({
-  type: Yup.string().equals(["ttl"]),
-  min_duration_ms: positiveInteger.default(defaultMinDurationMs),
-  min_duration_seconds: positiveInteger.default(
-    Math.ceil(defaultMinDurationMs / 1000),
-  ),
-  multiplier: positiveInteger.default(10),
-});
-
-export const durationStrategyValidationSchema = Yup.object({
-  type: Yup.string().equals(["duration"]),
-  duration: positiveInteger.default(24),
-  unit: Yup.string().test(
-    "is-duration-unit",
-    "${path} is not a valid duration",
-    value => !!value && durationUnits.has(value),
-  ),
-});
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 39]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This is dead code<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/metabase/metabase/43817?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->